Singularity/Singularity/Levels/BasicLevel.cs

```diff
@@ -65,9 +67,8 @@ private void StandardInitialization(ContentManager content)
             var platformBlankTexture = content.Load<Texture2D>("PlatformBasic");
             var platformDomeTexture = content.Load<Texture2D>("Dome");
             var mapBackground = content.Load<Texture2D>("backgroundGrid");
-            var libSans12 = content.Load<SpriteFont>("LibSans12");
 
-            PlatformFactory.Init(platformConeTexture, platformCylTexture, platformDomeTexture, platformBlankTexture, libSans12);
+            PlatformFactory.Init(platformConeTexture, platformCylTexture, platformDomeTexture, platformBlankTexture);
```

Comment:
Das hier und in den ganzen Plattformen ist zeuch vom Master, Änderungen von mir die du da Rückgängig machst .. ich weiß nicht ob das gedacht war von dir ... ?

Reply:
also die ganzen `libSans12`-geschichten und die PlatformInfoBox-dinger